src/main.ts

```diff
@@ -79,35 +219,25 @@ export default class MarkwhenPlugin extends Plugin {
 
 	//Credits to https://github.com/yuleicul/obsidian-ketcher on file operations
 	async createAndOpenMWFile(
-		filename: string,
+		filename?: string,
 		foldername?: string,
 		initData?: string
 	) {
-		const file = await this.createMWFile(filename, foldername, initData);
+		filename = filename ?? `Markwhen ${new Date()
+			.toLocaleString('en-US', { hour12: false })
+			.replace(/\//g, '-')
+			.replace(/:/g, '.')
+			.replace(/,/, '')}`;
```

Comment:
Can we just use the Obsidian methodology and  call them `Untitled` and not have the date in the filename?

Reply:
We can in theory use untitled but still, if user spams to open file, every time we would need to check the last untitled name in the folder and add 1 to that name. I would make a toggle for this when this update is passed so that we can open files with time and/or untitled